domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/TimelineEvent.java

```diff
@@ -1,15 +1,18 @@
 package uk.gov.hmcts.cmc.domain.models;
 
+import lombok.Builder;
 import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.NotBlank;
 
 import javax.validation.constraints.Size;
 
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
-@EqualsAndHashCode
-public class TimelineEvent {
+@EqualsAndHashCode(callSuper = true)
```

Comment:
Good Work @dharmendrak , converting to builder. We should start doing such small tech debts pushed in as part of PR